File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.cs
using System;

public class Solution {
    public int solution(int[] num_list) {        
        int mul = 1;
        int sum = 0;
        
        foreach(var i in num_list)
        {
            mul *= i;
        }
        
        foreach(var i in num_list)
        {
            sum += i;
        }
        sum = sum * sum;
        if(sum < mul) return 0;
        else return 1;
    }
}

File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) {
    int a1 = 0;
    int a2 = 1;
    for(int i : num_list)
    {
        a1 += i;
    }
    a1 = a1 * a1;
    for(int i : num_list)
    {
        a2 *= i;
    }
    return a1 > a2 ? 1 : 0;
}